charts/camunda-platform/values.yaml

```diff
@@ -146,6 +146,7 @@ global:
         redirectUrl: "http://localhost:8083"
 
 # Zeebe configuration for the Zeebe sub chart. Contains configuration for the Zeebe broker and related resources.
+# See https://docs.camunda.io/docs/self-managed/platform-deployment/docker/#zeebe
```

Comment:
This link is a bit confusing where Zeebe "docker" is mentioned given the line before it talks about Zeebe "chart".
Maybe we need to add more details about what's expected there.

Or moving it to a more specific section (not the top level key).